src/ml/neural_net/tf_compute_context.cpp

```diff
@@ -259,7 +259,8 @@ std::unique_ptr<compute_context> create_tf_compute_context() {
 
 // At static-init time, register create_tf_compute_context().
 // TODO: Codify priority levels?
-static auto* tf_registration = new compute_context::registration(
+static auto* tf_registration __attribute__((unused)) =
+  new compute_context::registration(
```

Comment:
What is this for?

Reply:
This tells the complier not to issues a warning about this variable being unused. This change was needed to compile on Ubuntu 16.04.

Reply:
Got it.